blog/2025/clio-2.5.0.md

```diff
@@ -0,0 +1,258 @@
+---
+category: 2025
+date: "2025-07-10"
+template: '../../@theme/templates/blogpost'
+seo:
+    title: Introducing Clio version 2.5.0
+    description: Version 2.5.0 of Clio, an XRP Ledger API server optimized for HTTP and WebSocket API calls, is now available. This release adds new features and bug fixes.
+labels:
+    - Clio Release Notes
+markdown:
+    editPage:
+        hide: true
+---
+# Introducing Clio version 2.5.0
+
+Version 2.5.0 of Clio, an XRP Ledger API server optimized for HTTP and WebSocket API calls, is now available. This release adds new features and bug fixes.
+
+## Amendment Support
+
+The following amendments have been introduced since Clio 2.4.1 and have transaction model changes. Clio 2.5.0 is built with `libxrpl` 2.5.0, which supports these amendments.
+
+- [Batch](https://xrpl.org/resources/known-amendments#batch)
+- [PermissionDelegation](https://xrpl.org/resources/known-amendments#permissiondelegation)
+- [PermissionedDEX](https://xrpl.org/resources/known-amendments#permissioneddex)
+- [TokenEscrow](https://xrpl.org/resources/known-amendments#tokenescrow)
+
+If these amendments are enabled and you have not upgraded Clio to 2.5.0 or newer, the ETL will be amendment blocked and new ledgers will not be processed.
+
+To check the current voting status of these amendments on Mainnet, see the [XRPL Amendments Dashboard](https://livenet.xrpl.org/amendments).
+
+## Install / Upgrade
+
+| Package  |
+| :------- |
+| [Clio Server Linux Release (GCC)](https://github.com/XRPLF/clio/releases/download/2.5.0/clio_server_Linux_Release_gcc.zip) |
+| [Clio Server macOS Release (Apple Clang 16)](https://github.com/XRPLF/clio/releases/download/2.5.0/clio_server_macOS_Release_apple-clang.zip) |
+
+For other platforms, please [build from source](https://github.com/XRPLF/clio/releases/tag/2.5.0). The most recent commit in the git log should be:
+
+```text
+Merge: b8b82e5d 8fcc2dfa
+Author: Sergey Kuznetsov <skuznetsov@ripple.com>
+Date:   Mon Jun 30 11:32:26 2025 +0100
+
+    chore: Commits for 2.5.0 (#2268)
+```
+
+### What's Changed
+
+See the [Full Changelog on GitHub](https://github.com/XRPLF/clio/compare/2.4.1...2.5.0).
+
+### Features
```

Comment:
I think it's more grammatically appropriate for features to read in present simple tense:

"Adds xyz" instead of "Add xyz". "Implements" vs "implements" etc.